common/centraldashboard/base/deployment.yaml

```diff
@@ -26,4 +26,6 @@ spec:
           value: $(userid-header)
         - name: USERID_PREFIX
           value: $(userid-prefix)
+        - name: PROFILES_KFAM_SERVICE_HOST
+          value: profiles-kfam
```

Comment:
Can we have the default value point to service address?
`kfam.kubeflow:8081`

Reply:
+1

Reply:
@avdaredevil - Why not change the default value in [server.ts](https://github.com/kubeflow/kubeflow/blob/840e24b5e86654d8e4a3596a3c91ffb2d198bca1/components/centraldashboard/app/server.ts#L12) so that it expects to resolve the `profiles-kfam` DNS name and has to overridden only when developing locally?

That would alleviate the need for this to be set in manifests.

Reply:
That's already tackled in kubeflow/kubeflow#3971 @prodonjs 

Sure @kunmingg @jlewi

Reply:
Port is a different argument and is already defaulted to 8081